README.md

```diff
@@ -147,6 +147,17 @@ For example, the `workbenchConfigFilePath` field might have the `anyFolder/myCon
 This means that the config file can be found by the path: `che-code/branding/anyFolder/myConfigFiles/workbench-config.json`
 - Currently, the [branding.sh](https://github.com/che-incubator/che-code/blob/main/branding/branding.sh) script is not run automatically when building this project. It needs to be integrated into the build process of the [downstream branded project or product](https://github.com/redhat-developer/devspaces-images/blob/devspaces-3-rhel-8/devspaces-code/build/scripts/sync.sh#L96).
 
+# Builds
```

Comment:
@nickboldt let's name this section to `Downstream builds` and mention the downstream-related information only.

Reply:
That would make your readme inconsistent with the other projects. https://github.com/eclipse/che/issues/21790

What's wrong with linking to the GH actions for your project ?

Reply:
I don't see any value in mentioning some of the Workflows there.
But I'm not against merging the PR as it is :slightly_smiling_face: